### PR TITLE
[zh] add blog docs-localized.md

### PR DESCRIPTION
**Pull request by shalk**

### CODE REVIEW

File: content/zh/blogs/2024/docs-localized.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+linkTitle: 多语言网站
+date: 2024-08-20
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco)
+issue: 4863
+sig: Comms
+default_lang_commit: aca2e07f0d3a6
+---
+
+我们很高兴地宣布，OpenTelemetry 网站现已提供多种语言版本！本地化团队已经开始将网站页面翻译成 [中文](/zh), [日语](/ja), [葡萄牙语](/pt), and [西班牙语](/es)
```
Comment (reyang) [inline content/zh/blogs/2024/docs-localized.md:12]:
```suggestion
我们很高兴地宣布，OpenTelemetry 网站现已提供多种语言版本！本地化团队已经开始将网站页面翻译成 [中文](/zh), [日语](/ja), [葡萄牙语](/pt), 以及[西班牙语](/es)
```

File: content/zh/blogs/2024/docs-localized.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+OpenTelemetry 项目已发展壮大，包括来自世界各地的贡献者和用户。提供多种语言的网站是确保每个人（无论其母语是什么）都能为项目做出贡献的重要一步。我们也很高兴这一举措让最终用户可以访问多语言文档，从而更轻松地学习和理解 OpenTelemetry。
+
+要以您的首选语言访问网站，请使用页面右上角的语言选择器。
+
+
```
Comment (reyang) [inline content/zh/blogs/2024/docs-localized.md:18]:
```suggestion

```

File: content/zh/blogs/2024/docs-localized.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+我们邀请您为此做出贡献。您可以在 [#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37) Slack 频道中找到其他本地化贡献者。如果您精通已支持的语言，您可以通过编写或审阅翻译来提供帮助。如果您的语言尚不受支持，并且您想帮忙翻译，[创建一个问题](<https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3CYOUR%20LANGUAGE%3E+(%3CYOUR+CODE%3E)+version+of+website+pages&body=%3C!--+Provide+github+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E>)。
+
+你的贡献会产生重要的影响！
```
Comment (reyang) [inline content/zh/blogs/2024/docs-localized.md:21]:
```suggestion
您的贡献会产生重要的影响！
```

**Review by reyang — APPROVED** [review]
LGTM with some non-blocking suggestions.

File: content/zh/blogs/2024/docs-localized.md (hunk 1 above)
Comment (my-git9) [inline content/zh/blogs/2024/docs-localized.md:12]: ","  ==> "、"

Comment (shalk) [reply content/zh/blogs/2024/docs-localized.md]: done

File: content/zh/blogs/2024/docs-localized.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+linkTitle: 多语言网站
+date: 2024-08-20
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco)
+issue: 4863
+sig: Comms
+default_lang_commit: aca2e07f0d3a6
+---
+
+我们很高兴地宣布，OpenTelemetry 网站现已提供多种语言版本！本地化团队已经开始将网站页面翻译成 [中文](/zh)、[日语](/ja)、[葡萄牙语](/pt)、以及[西班牙语](/es)
```
Comment (windsonsea) [inline content/zh/blogs/2024/docs-localized.md:12]:
```suggestion
我们很高兴地宣布，OpenTelemetry 网站现已提供多种语言版本！
本地化团队已经开始将网站页面翻译成[中文](/zh)、[日语](/ja)、[葡萄牙语](/pt)以及[西班牙语](/es)。
```

File: content/zh/blogs/2024/docs-localized.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+OpenTelemetry 项目已发展壮大，包括来自世界各地的贡献者和用户。提供多种语言的网站是确保每个人（无论其母语是什么）都能为项目做出贡献的重要一步。我们也很高兴这一举措让最终用户可以访问多语言文档，从而更轻松地学习和理解 OpenTelemetry。
```
Comment (windsonsea) [inline content/zh/blogs/2024/docs-localized.md:14]:
```suggestion
OpenTelemetry 项目已发展壮大，包括来自世界各地的贡献者和用户。
提供多种语言的网站是确保每个人（无论其母语是什么）都能为项目做出贡献的重要一步。
我们也很高兴这一举措让最终用户可以访问多语言文档，从而更轻松地学习和理解 OpenTelemetry。
```

File: content/zh/blogs/2024/docs-localized.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+要以您的首选语言访问网站，请使用页面右上角的语言选择器。
```
Comment (windsonsea) [inline content/zh/blogs/2024/docs-localized.md:16]:
```suggestion
要以您的首选语言访问网站，请使用页面右上角的语言选择菜单。
```

File: content/zh/blogs/2024/docs-localized.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+我们邀请您为此做出贡献。您可以在 [#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37) Slack 频道中找到其他本地化贡献者。如果您精通已支持的语言，您可以通过编写或审阅翻译来提供帮助。如果您的语言尚不受支持，并且您想帮忙翻译，[创建一个问题](<https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3CYOUR%20LANGUAGE%3E+(%3CYOUR+CODE%3E)+version+of+website+pages&body=%3C!--+Provide+github+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E>)。
```
Comment (windsonsea) [inline content/zh/blogs/2024/docs-localized.md:17]:
```suggestion
我们邀请您为此做出贡献。您可以在 [#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37) Slack
频道中找到其他本地化贡献者。如果您精通已支持的语言，您可以通过编写或审阅翻译来提供帮助。
如果您的语言尚不受支持，并且您想帮忙翻译，[请创建一个 Issue](<https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3CYOUR%20LANGUAGE%3E+(%3CYOUR+CODE%3E)+version+of+website+pages&body=%3C!--+Provide+github+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E>)。
```

**Review by windsonsea — COMMENTED** [review]
Wrap long lines to provide convenience for later update.

File: content/zh/blogs/2024/docs-localized.md (hunk 1 above)
Comment (windsonsea) [reply content/zh/blogs/2024/docs-localized.md]: Need not reply each comment. You can only click `Resolve conversation` if you have applied the suggestions.

**Review by windsonsea — APPROVED** [review]
Thanks, lgtm.

**Pull request merged**